xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
i'm prolly totally missing the point here, i thought the interactive flag in shells is the same with python and ipython REPL.